K2Bridge/Startup.cs

```diff
@@ -80,12 +81,22 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddTransient<ITranslator, ElasticQueryTranslator>();
 
+            var dynamicSamplePercentage = GetConfigOptional<double>("dynamicSamplePercentage");
+            if (dynamicSamplePercentage is < 0 or > 100)
```

Comment:
what does dynamicSamplePercentage=0 means?
that we are not treating this field as dynamic? maybe it should be > 0

Reply:
You're right, it's now >0